libvirt/utils_net.go

```diff
@@ -96,7 +117,7 @@ func (fws *fileWebServer) Start() error {
 
 	handler := http.NewServeMux()
 	handler.Handle("/", http.FileServer(http.Dir(dir)))
-	fws.server = &http.Server{Addr: fmt.Sprintf(":%d", fws.Port), Handler: handler}
+	fws.server = &http.Server{Addr: fmt.Sprintf("127.0.0.1:%d", fws.Port), Handler: handler}
```

Comment:
I assume that somehow this bug was related https://github.com/travis-ci/travis-ci/issues/8711? Are there any downsides to hardcode `127.0.0.1` (except that IPv6 doesn't work 😞)